File: app/src/main/java/com/vkpapps/soundbooster/interfaces/OnMediaPlayerChangeListener.java
package com.vkpapps.soundbooster.interfaces;

import android.media.MediaPlayer;

/**
 * @author VIJAY PATIDAR
 * */
public interface OnMediaPlayerChangeListener {
    void onChangeSong(String title, MediaPlayer mediaPlayer);

    void onPlayingStatusChange(boolean isPlaying);

    void onVolumeChange(float volume);
}

File: app/src/main/java/com/vkpapps/soundbooster/interfaces/OnMediaPlayerChangeListener.kt
package com.vkpapps.soundbooster.interfaces

import androidx.annotation.UiThread

/**
 * @author VIJAY PATIDAR
 */
interface OnMediaPlayerChangeListener {
    @UiThread
    fun onChangeSong(title: String)

    @UiThread
    fun onPlayingStatusChange(isPlaying: Boolean)

    @UiThread
    fun onVolumeChange(volume: Float)
}